### ISSUE TITLE
Webinterface: [ERROR] File not found: /etc/pihole/setupVars.conf

### ISSUE THREAD

**Issue by mdosch** [pi-hole/pi-hole#2190]
I installed pi-hole today on Debian 9 amd64. DNS is working and also reachable from other machines in the network but when trying to acces the Web-Interface I get the error "[ERROR] File not found: /etc/pihole/setupVars.conf".

The file /etc/pihole/setupVars.conf is existing and for testing purposes I even set permissions to 777 but to no avail. I found #1111 but that seems to be fixed long time ago.

Debug token: iq2z47b5s3

Feel free to ask if you need further output. Thanks.

**Comment by dschaper** [comment]
The debug script was able to read and parse the `setupVars.conf` file correctly, so it does exist and is readable, but it looks like `dnsmasq` is not able to read any files in `/etc/pihole`. Can you run `sudo systemctl --no-pager -l status dnsmasq.service` to see if there are any errors listed in the current status.

**Comment by dschaper** [comment]
Also the `setupVars.conf` looks to be missing upstream DNS servers, so that will need to be fixed for any DNS resolution to happen.

**Comment by mdosch** [comment]
The upstream servers where a temporary problem with my dnscrypt-proxy but they should not have affected web access.
I saw that there is a user pihole and www-data for pi-hole and the installed lighttp. After setting ownership to pihole:www-date for /etc/pihole and /var/www/html everything works as expected.

Sorry for bothering you as I was not aware I have to adjust permissions. Maybe this was caused by my very restrictive umask.
